ocaml/testsuite/tests/parsetree/locations_test.compilers.reference

```diff
@@ -266,12 +266,18 @@ Ptop_def
             expression (//toplevel//[2,1+4]..[2,1+35])
               Pexp_constraint
               expression (//toplevel//[2,1+25]..[2,1+35])
+                attribute "jane.erasable._builtin.param"
```

Comment:
All the additions here make me worry about performance cost here of the more elaborate encoding.

Reply:
Per [this comment](https://github.com/ocaml-flambda/flambda-backend/pull/1548#issuecomment-1662282983), the attribute encoding is now less heavy and this test's output is updated as such.